packages/dashboard/src/components/queryEditor/iotSiteWiseQueryEditor/modeledDataStreamQueryEditor/modeledDataStreamExplorer/modeledDataStreamTable/createModeledDataStreamColumnDefinitions.ts

```diff
@@ -14,13 +17,23 @@ export function createModeledDataStreamColumnDefinitions(): TableProps<
     {
       id: 'latestValue',
       header: 'Latest value',
-      cell: ({ latestValue }) => latestValue,
+      cell: ({ latestValue }) => {
+        if (latestValue && isNumeric(latestValue) && !Number.isInteger(latestValue)) {
```

Comment:
does the not integer check matter? I think the round function will not do anything if it is an integer since it only applies the toFixed precision on the decimal value

Reply:
thanks for the comment, I removed the integer check.